arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -12,65 +12,117 @@ This how-to provides step-by-step instructions for running an Orbit node on your
 
 - **RAM**: 8-16 GB
 - **CPU**: 2-4 core CPU (For AWS: t3 xLarge)
-- **Storage**: It depends on the Orbit chain and its traffic during the time.
+- **Storage**: This depends on the Orbit chain and its traffic overtime.
 
 ### Prerequisites
 
 - Latest Docker Image: <code>@latestNitroNodeImage@</code>
 
-### Required parameters
+## Required parameters
```

Comment:
nitpick: Adding the comment here, although it's more for previous headers.
We are using level 3 headers after the level 1 title, and then here we start using level 2 headers.
We should either change the previous header to level 2, or create a level 2 header before those.

Reply:
That's a good point.
TBH I don't always use MD headers incrementally since they are just templates for what needs to be outlined. In this article, prerequisites are not the "root" of the structure, but they are among the first elements that need to be communicated. Does it make sense?

Reply:
Interesting 🤔 
IMO, headers should outline the structure of the information of a page (with a specific order). If there's a subsection that doesn't seem to suit the specific header it should have, there might be something missing in that subsection (an introduction, or some content before that subsection?). Happy to discuss further though, I find your point of view interesting (hadn't thought about it).

In this specific case, I see your point in that the current content in prerequisites might not fit the place it currently lives in, and the page structure overall could be improved. Let me propose something: what about having a first section titled "Prerequisites" with the minimum hardware requirements, and the Nitro version needed? (no need for extra subtitles there). That prerequisites would feel more like an `h2` (at least from my point of view). What do you think?

Adding another note on headers, when moving MD headers to HTML, keeping them ordered also improves SEO in general as it helps search engines parse the information of the page correctly.

Reply:
I was unaware of the SEO implications of not using an incremental Header order; thanks for pointing this out.
I have no opinion on the best way to display a prerequisite paragraph. I just don't want it to steal the reader's attention from the main paragraphs. 
What header template would you propose for the `prerequisite` paragraph?

Reply:
Yeap, better. Thanks! 🙏